src/components/image/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/macro';
+
+const Background = styled.div`
+    width: ${(props) => props.width};
+    height: ${(props) => props.height};
+    max-width: ${(props) => props.maxWidth};
+    max-height: ${(props) => props.maxHeight};
+    background-image: url(${(props) => props.src});
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position-x: ${(props) => props.horizontalAlignment};
+    background-position-y: ${(props) => props.verticalAlignment};
+`;
+Background.displayName = 'Background';
+
+const Container = styled.div`
+    width: 100%;
+`;
+Container.displayName = 'Container';
+
+const Img = styled.img`
+    width: ${(props) => props.width};
+    max-width: ${(props) => props.maxWidth};
+    max-height: ${(props) => props.maxHeight};
+    flex: 0 0 auto;
+`;
+Img.displayName = 'Img';
+
+Img.defaultProps = {
+    width: '100%',
+    maxWidth: '100%',
+    maxHeight: '100%'
+};
+
+export class Image extends Component {
+    render() {
+        if (this.props.type === 'css') {
```

Comment:
I'd name this `this.props.type === 'css'` to be a bit more clear. This tripped me up a bit. 

I'd name the function like 
```
  isBackgroundImage() {
  return this.props.type === 'css'
}
```

and then do `if(this.isBackgroundImage) { ` etc

Reply:
I can change this